lib/src/js/value/color.dart

```diff
@@ -87,8 +87,10 @@ final JSClass colorClass = () {
     'toSpace': (SassColor self, String space) => _toSpace(self, space),
     'isInGamut': (SassColor self, [String? space]) =>
         _toSpace(self, space).isInGamut,
-    'toGamut': (SassColor self, [String? space]) =>
-        _toSpace(self, space).toGamut(),
+    'toGamut': (SassColor self, [String? space]) {
+      var originalSpace = self.space;
+      return _toSpace(self, space).toGamut().toSpace(originalSpace);
```

Comment:
> That's very plausible. Feel free to just mark those tests as todo for now and I'll reenable them as I get the implementation into shape.

One of the tests was failing because of this bug, but I added comments to the rest of the failing tests, which I think are all caused by implementation lagging spec changes.